crates/router/tests/connectors/coinbase.rs

```diff
@@ -1,6 +1,13 @@
+use std::marker::PhantomData;
```

Comment:
Can you revert this test file changes, Currently we are adding non critical testcases using [new man postman ](https://github.com/juspay/hyperswitch/wiki/A-better-storage-format-for-postman-collections). It will be quite effort for you to follow this, Instead we can internally assign this to someone from team.